node.go

```diff
@@ -135,53 +53,65 @@ func (n *SphinxNode) PrefixFreeDecode(s []byte) (int, []byte, []byte) {
 	return Failure, nil, nil
 }
 
-// Unwrap unwraps a layer of encryption from a sphinx packet
-// and upon success returns an UnwrappedMessage, otherwise an error.
-func (n *SphinxNode) Unwrap(packet *SphinxPacket) (*UnwrappedMessage, error) {
+// UnwrappedMessage is produced by SphinxNode's Unwrap method
+type UnwrappedMessage struct {
+	ProcessAction int
+	Alpha         []byte
```

Comment:
comments explaining these fields would be useful.  In particular,is 'ProcessAction' really an enum? can alpha/beta/gamma/delta be explained as message content or somehow not in a way that sheds to light on their meaning? Is NextHop a public key, a server identifier, something else?